src/NzbDrone.Core.Test/HealthCheck/Checks/DownloadClientRootFolderCheckFixture.cs

```diff
@@ -68,7 +68,7 @@ public void should_return_downloads_in_root_folder_if_downloading_to_root_folder
         {
             GivenRootFolder(_downloadRootPath);
 
-            Subject.Check().ShouldBeWarning(wikiFragment: "downloads_in_root_folder");
+            Subject.Check().ShouldBeWarning(wikiFragment: "downloads-in-root-folder");
```

Comment:
We don't need to check the fragment, we only care that there is a warning as the result. I've removed from develop.

Reply:
removed this commit and rebased as you've fixed in develop